HDEV61_validation.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
```

Comment:
Please don't include this script in your final commit. Prefer unit/integration tests (that do not require usage of the live anthropic apis) to run.